woocommerce/woocommerce-ios
# About screen add system images for accessory view

## Description


- "Added accessoryViewSystemImage to AboutItem" in https://github.com/Automattic/AutomatticAbout-Swift/pull/9
- Using that change to add icons next to rate and share rows
- renamed "Twitter" to "X"

## Testing instructions


- Open Menu
- Open Settings
- Scroll to "About the app" and tap "WooCommerce"
- Check that "Rate us" and "Share with Friends" have icons next to it
- Check that "Instagram" has "@woocommerce" for the subtitle
-

## Screenshots


| Before      | After |
| ----------- | ----------- |
| ![Simulator Screenshot - iPhone 15 Pro - 2024-03-12 at 16 44 32](https://github.com/woocommerce/woocommerce-ios/assets/6242034/ff6c0522-6e92-4bf0-97ee-a2896d3fdb4d)      | ![Simulator Screenshot - iPhone 15 Pro - 2024-03-15 at 17 16 26](https://github.com/woocommerce/woocommerce-ios/assets/6242034/998bb647-568d-4e0e-880c-c51ebc9fb21a)      |




---
